configuration/appframework/coreapps/appointments_app.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "coreapps.appointmentsApp",
+    "instanceOf": "coreapps.template.findPatient",
```

Comment:
These apps should only be instances of the coreapps.template.findPatient app if it makes sense to first have a patient search when you enter it, and then for the result of that patient search to take you to a patient-specific page.  In most of the cases I see here and below, this is not the case.  These should instead follow the model of things like the home_configuration_extension.json - eg the imbemr.myAccountHomePageLink - where you are just configuring a link on the home page to a specific page in the application, not configuring something that requires a patient search.  I understand maybe this is actually incorrect in the coreapps code that we copied, but we can rectify that here.

Reply:
Ohh I understand that, I will change it. Thank you!